src/components/PlayersList.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+import React from 'react';
+
+
+function PlayersList({ players, handlePlayerOnClick }) {
+  return (
+      <ul onClick={handlePlayerOnClick} style={{ "listStyleType": "none", "display": "inline" }}>
+        {Object.entries(players).map(item => {
```

Comment:
since you're not using the key can prob just do this
Object.values(players).map({id: itemID, username} => (
            <li key={`${itemID + username}`} id={itemId}>{username}</li>
          )

Reply:
Great catch! I should have thought about it further and `Object.values` is much better.